### PR TITLE
updated fizzbuzz file

### PR DESCRIPTION
**Pull request by KSKRLIFE**

### CODE REVIEW

**Comment by rohan-singh1** [comment]
The code is correct. However, my suggestion would to indent the code; that would significantly improve its readability.

**Comment by KSKRLIFE** [comment]
> The code is correct. However, my suggestion would to indent the code; that would significantly improve its readability.

Thank you so much, will improvise it.

File: hw1/01-fizzbuzz.js [hunk 1]
```diff
@@ -9,8 +9,30 @@ Use console.log() to write the proper output to the command line.
 
 **/
 
-const fizzbuzz = () => {
-  // Add your code here
+const fizzbuzz = () =>
+{
+
+for(let i =1; i<=100; i++)
+{
```
Comment (rohan-singh1) [inline hw1/01-fizzbuzz.js:16]: IDEs will auto indent the code too.

**Review by rohan-singh1 — COMMENTED** [review]
Better to indent the code, as it will improve its readability and reduce chances of introducing bugs.

**Review by TimmyYoung — APPROVED** [review]
The code looks like the right implementation. It took me a second to remember from taking numerical computations that if two factors evenly divide an integer, than their product also evenly divides larger products from those two factors. I was thinking this may skip some integers and produce bugs, but after doing some digging it appears to be correct.

I do think style changes would help aid your code readability. 22 lines is rather large for a small code exercise. Lining up brackets is helpful in some cases, but it tends to quickly add to the length of a file in my opinion.

**Review by Al1999 — COMMENTED** [review]
Hi Sai, 

Your code looks good. It needs to be indented in some areas. Also, you can change comparisons  == to === . 
but overall, good job :)

**Comment by SwaroopaSK** [comment]
Hi Sai, The code looks good, indentation will increase the readability.

**Comment by JeremyIV** [comment]
For future assignments, please incorporate the feedback you get (e.g. indent your code) into your pull request before merging it into main.

**Comment by KSKRLIFE** [comment]
> For future assignments, please incorporate the feedback you get (e.g. indent your code) into your pull request before merging it into main.

Sure. Will do that. Thank you.

**Pull request merged**